provd/example.config.yaml

```diff
@@ -1,2 +1,2 @@
 Paths:
-  Socket: "/run/test.sock"
+  Socket: "/home/matt/test.sock"
```

Comment:
Was that intentional? I was wondering why my local test setup broke and found this :D

Reply:
Haha, no, that was not intentional :sweat_smile: I'll revert that back!

Reply:
I did the [same](https://github.com/canonical/ubuntu-desktop-provision/compare/a28f5bbc95b932698c3ebee26f94bbc457ec48f4..0bbaa6168c5b3ddedcd039e1091755f890cc9dd6) now.. :grimacing: